build.go

```diff
@@ -20,7 +21,42 @@ type SBOMGenerator interface {
 	Generate(path string) (sbom.SBOM, error)
 }
 
+// Configuration enumerates the environment variable configuration options
+// that govern the buildpack's behaviour.
+type Configuration struct {
```

Comment:
Just a suggestion so feel free to take it or leave it, but would it be better to have it in a separate file? The config is used in both detect and build so might just make sense in a separate configuration file